contracts/get_contract_metadata/assembly/proto/getcontractmetadata.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package getcontractmetadata;
+
+import "koinos/options.proto";
+import "koinos/chain/chain.proto";
+
+// entry points
+
+// @description Get a contract's metadata
+// @read-only true
+message get_contract_metadata_arguments {
+  bytes contract_id = 1 [(koinos.btype) = CONTRACT_ID];
+}
+
+message get_contract_metadata_result {
+  koinos.chain.contract_metadata_object contract_metadata = 1;
```

Comment:
The field should be named `value`. See other getter system calls. This will become important when implementing in C++ as it is required by system call macros to define a native C++ implementation.

```proto3
message get_account_nonce_arguments {
   bytes account = 1 [(btype) = ADDRESS];
}

message get_account_nonce_result {
   bytes value = 1;
}
```